spec/support/service_offering_topological_service_offering_helper.rb

```diff
@@ -10,7 +10,11 @@ def fully_populated_service_offering
       :documentation_url        => "http://test.docs.io",
       :support_url              => "800-555-TEST",
       :distributor              => "Red Hat Inc.",
-      :service_offering_icon_id => "998"
+      :service_offering_icon_id => "998",
+      :extra                    => {
+        :type           => "Job Template",
```

Comment:
Possibly we should match the Tower object return as snake case `"job_template"`

Reply:
Agreed.  Current values for type are either `job_template` or `workflow_job_template`